src/forklift/change_detection.py

```diff
@@ -87,6 +103,14 @@ def update(self, crate):
         return (status, None)
 
 
+def _has_globalid_field(table):
+    '''table: string - path to a feature class
+
+    returns a boolean indicating if the table has a GlobalId field
+    '''
+    return len(arcpy.ListFields(table, field_type='GlobalID')) > 0
```

Comment:
**question**: should we be using the [globalIDFieldName](https://pro.arcgis.com/en/pro-app/latest/arcpy/functions/gdb-table-properties.htm) from a table describe?

Reply:
After looking at those docs, I noticed a `hasGlobalID` prop which seems like an even better solution. Thanks for the prompt to look at them.